### PR TITLE
Rendering label text

### PR DESCRIPTION
**Pull request by lajunek**
Goal: Users shall get precise information about handling automated labeling. There is a choice between "Always render labels as paths" and "Always render labels as text".

Ticket(s): #3239

- [x] Backport to LTR documentation is required

### CODE REVIEW

**Comment by lajunek** [comment]
@DelazJ @SrNetoChan can you implement the new screenshot here? I'm using windows and so the look of the windows would be different

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -1231,15 +1231,16 @@ options:
...
-  drawn (and exported) as either proper text objects OR as paths only. If they
-  are exported as text objects then they can be edited in external applications
-  (e.g. Inkscape) as normal text. BUT the side effect is that the rendering
-  quality is decreased, AND there's issues with rendering when certain text settings
-  like buffers are in place. That's why drawing as outlines is recommended. Note
-  that when :ref:`exporting a layout to svg <export_layout_svg>` there's actually
-  an override for this setting - so you can leave the project rendering as outlines
-  but for a :file:`.svg` export export the labels as text.
+* :guilabel:`Text rendering`|selectString| : controls whether text labels are
+  drawn (and exported) as either proper text objects (:guilabel:`Always Render Labels as Text`)
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]:
```suggestion
  drawn (and exported) as either proper text objects
  (:guilabel:`Always Render Labels as Text`)
```

Comment (havatv) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: The line is a bit long.

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 2]
```diff
@@ -1231,15 +1231,16 @@ options:
...
+  OR as paths only (:guilabel:`Always Render Labels as Paths (recommended)`). 
+  If they are exported as text objects then they can be edited
+  in external applications (e.g. Inkscape) as normal text. BUT the side effect is that
+  the rendering quality is decreased, AND there's issues with rendering when
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]:
```suggestion
  the rendering quality is decreased, and there are issues with rendering when
```

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 3]
```diff
@@ -1231,15 +1231,16 @@ options:
...
+  certain text settings like buffers are in place. That's why rendering as paths
+  is recommended. Note that when :ref:`exporting a layout to svg <export_layout_svg>`
+  there's actually an override for this setting - so you can leave the project
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]:
```suggestion
  there's an override for this setting.
  So you can have `Always Render Labels as Paths` for the project, and choose 
```

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 4]
```diff
@@ -1231,15 +1231,16 @@ options:
...
-  an override for this setting - so you can leave the project rendering as outlines
-  but for a :file:`.svg` export export the labels as text.
+* :guilabel:`Text rendering`|selectString| : controls whether text labels are
+  drawn (and exported) as either proper text objects (:guilabel:`Always Render Labels as Text`)
+  OR as paths only (:guilabel:`Always Render Labels as Paths (recommended)`). 
+  If they are exported as text objects then they can be edited
+  in external applications (e.g. Inkscape) as normal text. BUT the side effect is that
+  the rendering quality is decreased, AND there's issues with rendering when
+  certain text settings like buffers are in place. That's why rendering as paths
+  is recommended. Note that when :ref:`exporting a layout to svg <export_layout_svg>`
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]:
```suggestion
  is recommended. Note that when :ref:`exporting a layout to svg <export_layout_svg>`,
```

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 5]
```diff
@@ -1231,15 +1231,16 @@ options:
...
+  there's actually an override for this setting - so you can leave the project
+  rendering as paths but for a :file:`.svg` export export the labels as text.
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]:
```suggestion
  `Always Render Labels as Text` for :file:`.svg` export.
```

**Comment by havatv** [comment]
> @DelazJ @SrNetoChan can you implement the new screenshot here? I'm using windows and so the look of the windows would be different

@lajunek, I don't think there is a policy on OpSys, so it should be no problem that we mix Linux / OS X, Windows screenshots.  So I say - just go ahead and use windows screenshots.

**Comment by DelazJ** [comment]
Isn't the screenshot already up-to-date?

File: source/docs/user_manual/working_with_vector/vector_properties.rst (hunk 1 above)
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: @nyalldawson is this setting used for exporting? And where? I can see that the print layout has its own setting, the save map as pdf also does. What's then the use case of the "text rendering" in the placement engine?

Comment (nyalldawson) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: Hm, I guess it predated the option for "save map as pdf" to control this setting. But in any case, it acts as the default value for this setting for the project...

**Comment by DelazJ** [comment]
> color for unplaced labels …
6a07ec9
There is no option for coloring unplaced labels anymore

@lajunek Is this in 3.10?

**Comment by lajunek** [comment]
@DelazJ @SrNetoChan can you implement the new screenshot here? I'm using windows and so the look of the windows would be different

**Comment by lajunek** [comment]
@DelazJ actually I found this in 3.4.5 - Madeira

**Comment by lajunek** [comment]
@DelazJ I was talking just about the content of the screenshot, not the option to color unplaced labels in general in qgis 3.4

**Comment by DelazJ** [comment]
The issue is that we are documenting 3.10, not 3.4. So the screenshot should be coherent with that version.

**Comment by lajunek** [comment]
@DelazJ so when a `[x] backport to LTR documentation is required` is set in the PRs this will refer to QGIS 3.10?

**Comment by lajunek** [comment]
@DelazJ   when dealing with the master of qgis/QGIS-Documentation: is this directed to [https://docs.qgis.org/testing/](url) ?

**Comment by DelazJ** [comment]
> @DelazJ when dealing with the master of qgis/QGIS-Documentation: is this directed to https://docs.qgis.org/testing/ ?

Yes! And the features described in testing are of QGIS 3.10.  testing will be officially replaced with 3.10 in February.
> @DelazJ so when a [x] backport to LTR documentation is required is set in the PRs this will refer to QGIS 3.10?

No! the LTR refers to 3.4. And IMO, this checkbox should be ignored. So close to the end of life of 3.4, we should stop backporting to that version because it repeatedly breaks translators efforts and wastes the small time/energy we have.

**Pull request merged**